.circleci/config.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           name: Run integration tests
           command: |
             cp .env.integration-tests .env
-            make refresh SKIP_DUMMY=1 PYTEST_SENTRY_DSN=$PYTEST_SENTRY_DSN up_prod_detached integration_test_nimbus PYTEST_ARGS="$PYTEST_ARGS"
```

Comment:
Yeah don't take this out, this'll create a bunch of experiments in a bunch of states including various live states and they'll get synced into remote settings and that could weirdly screw up the integration tests.  I'll just add a step to add projects for the integration tests https://github.com/mozilla/experimenter/issues/8198

Reply:
Here use this https://github.com/mozilla/experimenter/pull/8200